core/common/src/main/java/alluxio/exception/PreconditionMessage.java

```diff
@@ -64,6 +64,7 @@
       "Acquire Remote Worker Client cannot not be called with local hostname"),
   S3_BUCKET_MUST_BE_SET("The %s system property must be set to use the S3UnderStorageCluster"),
   TTL_ONLY_FOR_FILE("TTL can only be set for files"),
+  TTL_FILE_SHOULD_LESS_THAN_PARENT_DIRECTORY("The value of TTL should less than parent directory"),
```

Comment:
Can you explain a bit why we don't need to guarantee file's ttl should less than parent directory's ttl?